Vic2ToHoI4/Data_Files/cultureGroupToGraphics.txt

```diff
@@ -19,7 +19,7 @@
 # THE SOFTWARE.
 
 # Those with "#Revise" after it refers to generic portraits that should get a replacement
-# Currently 150 needed
+# Currently 238 needed
```

Comment:
It looks like your changes were applied to a very old version of this file, wiping out all your earlier work.

Reply:
Solved